src/options/modules/CustomOptionTriggers.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * This modules contains the custom triggers for some options that are added.
+ *
+ * @module modules/CustomOptionTriggers
+ */
+
+import * as AutomaticSettings from "/common/modules/AutomaticSettings/AutomaticSettings.js";
+
+import { COMMUNICATION_MESSAGE_TYPE } from "/common/modules/data/BrowserCommunicationTypes.js";
+
+
+/**
+ * Requests the permission for autocorrect settings.
+ *
+ * @private
+ * @param  {Object} optionValue
+ * @param  {string} [option]
+ * @param  {Object} [event]
+ * @returns {Promise}
+ */
+function applyAutocorrectPermissions(optionValue, option, event) {
+    // trigger update for current session
+    browser.runtime.sendMessage({
+        "type": COMMUNICATION_MESSAGE_TYPE.AUTOCORRECT_BACKGROUND,
+        "optionValue": optionValue
+    });
+}
+
+
+/**
+ * Binds the triggers.
+ *
+ * This is basically the "init" method.
+ *
+ * @function
```

Comment:
```suggestion
 * @function
```

(likely a legacy thing I've once stopped doing. That is obvious here.)

Reply:
Would you like me to remove this line?

Reply:
It's not necessary. JSDoc knows this is a function.

Reply:
OK, I will remove this line in my next commit. I copied it verbatim from what you had in the Awesome Emoji Picker, so I can also remove it from there.

Reply:
Yeah, it's legacy code there…